### PR TITLE
changed for abs(eta)<2.1

### PR DESCRIPTION
**Pull request by rchudasa**

### CODE REVIEW

File: analysis/caloNoiseStudy/CaloNoiseHelpers.h [hunk 1]
```diff
@@ -19,13 +19,8 @@ enum EDet { kEB, kEE, kHB, kHE, kHFm, kHFp, nDets };
 vector<string> detNames = { "EB", "EE", "HB", "HE", "HFm", "HFp", "unknown" };
 
 vector<pair<double,double>> detLimits = {
-//  {0    , 1.479 }, // EB
```
Comment (jniedzie) [inline analysis/caloNoiseStudy/CaloNoiseHelpers.h:22]: Please pull my changes in noise thresholds calculation.

File: analysis/caloNoiseStudy/getCaloNoiseHistograms.cpp [hunk 2]
```diff
@@ -19,7 +19,7 @@ vector<string> allowedTriggers = {
 int main(int argc, char** argv)
 {
   if(argc!=3){
-    cout<<"Usage: ./getCaloNoiseHistograms inputPath outputPath"<<endl;
```
Comment (jniedzie) [inline analysis/caloNoiseStudy/getCaloNoiseHistograms.cpp:22]: It seems that also here you don't have the latest changes.

File: analysis/condor/runEfficienciesQED.sh [hunk 3]
```diff
@@ -1,32 +1,32 @@
 #!/bin/bash
 
-configPath="/afs/cern.ch/work/j/jniedzie/private/LightByLight2018/analysis/configs/efficiencies.md"
```
Comment (jniedzie) [inline analysis/condor/runEfficienciesQED.sh:3]: How about you create a variable storing base path, such that we don't have to change it everywhere in the script?

Comment (rchudasa) [reply analysis/condor/runEfficienciesQED.sh:3]: Incorporated all changes requested.

**Review by jniedzie — APPROVED** [review]

**Pull request merged**